### PR TITLE
style: landing page

### PR DESCRIPTION
**Pull request by buddy-web3**
@melissasamworth please note that Advanced Content follows the Acceptance Criteria, images must be added by editors in Coda

### CODE REVIEW

**Comment by Aprillion** [comment]
(side note: the editors are writers of "copy" == copy writers... copyright is not totally unrelated, but people writing for stampy are granting a CC license to the legal rights about copy as per the copyright link from our footer https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Copyright_su79L#_luPMa, they are not copyright holders, they are "just" the authors 😓)

**Comment by buddy-web3** [comment]
@Aprillion thank you for the very necessary clarification

File: app/components/ContentBox/box.css [hunk 1]
```diff
@@ -41,3 +41,13 @@
   border-color: var(--colors-teal-200);
   text-decoration: none;
 }
+
+.content-box-table.main img {
+  width: fit-content;
+  height: -webkit-fill-available;
```
Comment (Aprillion) [inline app/components/ContentBox/box.css:47]: does this work in Firefox too?

File: app/components/ContentBox/index.tsx [hunk 2]
```diff
@@ -22,14 +29,15 @@ export const ContentBox = ({title, action, actionTitle, children, className}: Co
           {actionTitle}
         </Button>
       </div>
-      <div className="content-box-table">{children}</div>
+      <div className={classNameTable}>{children}</div>
```
Comment (mruwnik) [inline app/components/ContentBox/index.tsx:32]: why not just wrap the child image in this?

**Review by mruwnik — APPROVED** [review]

**Pull request merged**